jme3-examples/src/main/java/jme3test/gui/TestRtlBitmapText.java

```diff
@@ -32,34 +32,67 @@
 package jme3test.gui;
 
 import com.jme3.app.SimpleApplication;
-import com.jme3.font.BitmapFont;
-import com.jme3.font.BitmapText;
-import com.jme3.font.LineWrapMode;
-import com.jme3.font.Rectangle;
+import com.jme3.app.StatsAppState;
+import com.jme3.font.*;
+import com.jme3.math.ColorRGBA;
 
 /**
  * Test case for JME issue #1158: BitmapText right to left line wrapping not work
  */
 public class TestRtlBitmapText extends SimpleApplication {
 
     // A right to left text.
-    final private String text = ".text left to right test a is This";
+    private String text = " check check check linelenght all for  possible .text left to right test a is This";
+    String text2 = "to right test a is This text left.";
+    BitmapFont fnt;
+
 
     public static void main(String[] args) {
         TestRtlBitmapText app = new TestRtlBitmapText();
         app.start();
     }
-
+    BitmapText txt;
     @Override
     public void simpleInitApp() {
-        BitmapFont fnt = assetManager.loadFont("Interface/Fonts/Default.fnt");
+        float H = 500;
+        float X = 400;
```

Comment:
Names of local variables should begin with a lowercase letter.

Reply:
Fixed.